namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-legal-adviser-api-staging/resources/ecr.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "ecr-repo-api" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=2.1"
```

Comment:
Please update this to use the latest version of the module (i.e. `ref=3.1`) You can find this by visiting the repo URL and checking for the latest entry in `tags`